back/app/models/initiative.rb

```diff
@@ -128,6 +128,15 @@ class Initiative < ApplicationRecord
       .where(initiative_status_changes: { initiative_status: InitiativeStatus.find_by(code: 'proposed') })
   end)
 
+  scope :activity_after, lambda { |time_ago|
+    joins(:initiative_status_changes)
+      .where(
+        'initiative_status_changes.initiative_status_id IN (?) AND initiative_status_changes.created_at > ?',
```

Comment:
Should we include comments and reactions here the same as for ideas for consistency? Not necessary of course, it's already a good improvement.